OOP/Assignment 1/OOPQuestion1/Mechanic.java

```diff
@@ -0,0 +1,41 @@
+public class Mechanic extends People
+{
+
+	private String type;
+	private String rate;
+
+	public void setRate(String r)
+	{
+		try
+		{
+			int val = Integer.parseInt(r);
+			this.rate = r;
+		}
+		catch (Exception e)
+		{
+			this.rate = "0";
+		}
+
+	}
+
+	public String getRate()
+	{
+		return this.rate;
+	}
+
+	public String getType()
+	{
+		return this.type;
+	}
+
+	boolean isAvailable;
+
+	public Mechanic(String mechId, String mechName, String mechType, String mechRate)
+	{
+		mechID = mechId;
```

Comment:
1. Create class of People    People p = new People();
2. p.setMechId(mechId);
3. p.getMechId()

Else remove the class People and move the code in Mechanic